02.CSS_manipulation/src/cssPropTs.ts

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const cssProp = (element: HTMLElement,
```

Comment:
isn't it more "pretty" to have something like
(
  element: HTMLElement,
  cssProperty: string | { [index: string]: string },
  value?: string,
)
with this it's also easier to add/remove new params.